plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -23,6 +23,8 @@ import {
   getMetricLabel,
   getNumberFormatter,
 } from '@superset-ui/core';
+import { EChartsOption, SeriesOption } from 'echarts';
```

Comment:
Is there something like `BoxplotSeriesOption`?

Reply:
Done!